### PR TITLE
Ac 1596 user management response handling

### PR DESCRIPTION
**Pull request by ABausG**
* Adjust handling of different scenarios
  * Login with valid credentials but user not part of the group: Ask to join group
  * Register with valid credentials and user is already part of the group: Login User
  * Register with email that is already registered: Hint that mail with confirmation link has been sent

### CODE REVIEW

File: packages/apptive_grid_user_management/lib/src/translation/l10n/translation_de.dart [hunk 1]
```diff
@@ -7,73 +7,86 @@ class ApptiveGridUserManagementLocalizedTranslation
...
   @override
-  String get actionCancel => "Abbrechen";
+  String get actionCancel => 'Abbrechen';
   @override
-  String get actionOk => "OK";
+  String get actionOk => 'OK';
   @override
   String requestResetPasswordSuccess(String email) =>
-      "Wir haben einen Link zum zurücksetzen an $email gesendet.";
+      'Wir haben einen Link zum zurücksetzen an $email gesendet.';
```
Comment (Nicoo12345) [inline packages/apptive_grid_user_management/lib/src/translation/l10n/translation_de.dart:63]: ...zum **Z**urücksetzen... 😉

**Comment by soeren-schmaljohann-2denker** [comment]
Why exactly does the user need to see and even confirm the 'join group' step. This feels like something that should be done automatically and maybe even without the user seeing anything about it, since it really doesn't concern him.

**Comment by ABausG** [comment]
> Why exactly does the user need to see and even confirm the 'join group' step. This feels like something that should be done automatically and maybe even without the user seeing anything about it, since it really doesn't concern him.

I added the step because I think it would be weird for users if they think they log in but then get to a "we have send you a link to confirm" page. That way they actively connect their account with the new app which I think feels more deliberate.
Maybe @jepedajah can chime in on this?

**Comment by Jepedajah** [comment]
> > Why exactly does the user need to see and even confirm the 'join group' step. This feels like something that should be done automatically and maybe even without the user seeing anything about it, since it really doesn't concern him.
> 
> I added the step because I think it would be weird for users if they think they log in but then get to a "we have send you a link to confirm" page. That way they actively connect their account with the new app which I think feels more deliberate. Maybe @Jepedajah can chime in on this?

I think the message is a good way to lead the user through our multiple instances of apptivegrid user components. Otherwise it seems a bit weird, why the user gets redirected to a verification mail after login.
It can stay the way it is

**Review by soeren-schmaljohann-2denker — APPROVED** [review]

**Pull request merged**